### ISSUE TITLE
Throw causes entire script to exit if selected blocks don't exist

### ISSUE THREAD

**Issue by mattclough1** [dixonandmoe/rellax#68]
https://github.com/dixonandmoe/rellax/blob/f3099f9b2a3b153f3d4f4838c438b08a66d2bb18/rellax.js#L95

Is this intentional behavior? I can't help but feel like maybe it should just `console.warn` and return. In my case, the module is included in bundle, and since some pages don't have the selected elements, it causes all the JS to exit.

**Comment by moeamaya** [comment]
Can you instantiate rellax only on the pages you need?

**Comment by mattclough1** [comment]
@moeamaya yeah, I just went ahead and checked whether the element was available before instantiating... depending on how someone is using Rellax, an exception is probably appropriate. In my case it's embellishment. Disregard! 😅

**Comment by moeamaya** [comment]
Hey I know how it goes when you're trying to ship your site, so no worries. 

I think i am going to keep the `throw` because I think it's best practice and breaking JS is maybe a feature in this case.

**Comment by CS-beglnner** [comment]
Similar issue without bundling

**Comment by KeironLowe** [comment]
@moeamaya Respectfully, I disagree with throwing an exception. Rellax is a nice to have and typically isn't required for the page to work so throwing an exception and stopping the entire script from executing doesn't make sense to me. I think throwing a warning instead would be more appropriate.

**Comment by moeamaya** [comment]
@KeironLowe How dare you say Rellax is "a nice to have" it's as critical as babel 😜

But okay I hear you all, and I recant my original positions. If anyone wants to make a PR happy to merge without the blocking exceptionl

**Comment by KeironLowe** [comment]
@moeamaya Some would say even more critical! Thanks, I've just submitted a PR.
